src/blinding.cpp

```diff
@@ -526,6 +593,22 @@ LIBSESSION_C_API bool session_blind25_sign(
     }
 }
 
+LIBSESSION_C_API bool session_blind_version_sign(
+        const unsigned char* ed25519_seckey,
+        CLIENT_PLATFORM platform,
+        size_t timestamp,
+        unsigned char* blinded_sig_out) {
+    try {
+        auto result = session::blind_version_sign(
+                {ed25519_seckey, 64}, static_cast<Platform>(platform), timestamp);
+        auto sig = result;
```

Comment:
```suggestion
        auto sig = session::blind_version_sign(
                {ed25519_seckey, 64}, static_cast<Platform>(platform), timestamp);
```
(avoids unintentionally copy)

Reply:
Similar to the above, looks like I just copied this from the blind_15/25 functions which I wrote a while back that was also doing this - have updated them as well